### DISCUSSION TITLE
Adding Linea mainnet support

### DISCUSSION THREAD

**Discussion by Akaryatrh** [wagmi-dev/references#386] [General]
Hello 🙂 

I'm part of ConsenSys organisation, and [as it has been done for Linea testnet](https://github.com/wagmi-dev/references/pull/214), we now would like to add support for Linea mainnet.
Is there any specific process to follow, or should we just open a new PR?

Thanks in advance for support!

**EDIT**: RPC endpoints for mainnet will be available in the upcoming weeks.

**Comment by Akaryatrh** [comment]
FYI, I opened a draft PR here https://github.com/wagmi-dev/references/pull/390

**Reply by Akaryatrh** [reply]
Side question @jxom @tmm would you be ok if `wagmi` was listed as part of Linea's ecosystem on [linea.build](https://linea.build/) website? If so, would you have a logotype to provide?

**Comment by Akaryatrh** [comment]
Also, we would like to provide you a (temporary) whitelisted Infura key to try Linea mainnet before launch date. Any way to contact one of you privately?

**Comment by dawnseeker8** [comment]
hi @jxom @tmm,  `linea` mainnet will launch soon, Is it ok if `wagmi` listed in Linea's ecosystem on [linea.build](https://linea.build/) website.

**Reply by dawnseeker8** [reply]
hi, @liho00 thank you for your reply, would you have a logotype to provide?

**Reply by liho00** [reply]
@dawnseeker8 yes i do have, shall I get your email in order to provide you the logo assets?

**Reply by dawnseeker8** [reply]
@liho00 can you send to leo.wang@consensys.net?, thank you. very much

**Comment by Akaryatrh** [comment]
FYI @jxom & @tmm I moved the PR to a ready state as explorers and bridge are now available.

**Comment by Akaryatrh** [comment]
Looks like PR is merged, thx a lot @jxom!

Any comment on this btw?
>would you be ok if wagmi was listed as part of Linea's ecosystem on [linea.build](https://linea.build/) website? If so, would you have a logotype to provide?

**Reply by tmm** [reply]
go ahead! light and dark vector logos here: https://github.com/wagmi-dev/.github/tree/main/content

**Reply by Akaryatrh** [reply]
Great, thanks a lot! 👍
